modules/ocf_desktop/files/stats/update.sh

```diff
@@ -4,12 +4,8 @@
 [ "$(imvirt 2>/dev/null)" == "Physical" ] || exit 0 # only count hours on desktops
 
 CUR_USER=$(who | awk '$5 == "(:0)" { print $1 }')
-DATA="state=inactive"
```

Comment:
Doesn't removing this mean that it will rely on the 3 minute timeout to close all sessions now? That seems like something that's not great compared to sending an update saying that nobody is logged in when the session has ended. This also means that some sessions might overlap if someone logs out and then someone quickly logs in, right?

Reply:
I noticed that when a new session is made, the old one is closed, so there won't be overlaps, but I believe the 3 minute thing is still a problem when nobody logs in after a user logs out.

Reply:
I agree it would be nicer to eagerly clean up sessions when they are known to have ended.

Reply:
I think shichenh would like eagerly cleaned up sessions as well.